README.md

```diff
@@ -1,23 +1,28 @@
 # caus-python
 Python implementation of CAUS
 
-## Requirements ##
-### Python Libraries ###
-kuberentes python client: https://github.com/kubernetes-client/python
-install with 'pip install kubernetes' onto machine or into a fitting virtual env
-
-Prometheus Api Client
-https://pypi.org/project/prometheus-api-client/
-pip install prometheus\_api\_client
-
-### Running Services ###
-Prometheus for Kubernetes; Metrics need to be accessible from inside the cluster (see monitoring module code)
-Kafka for Kubernetes
-
-### Kubernetes Deployment ###
-The caus-deployment.yaml has to be altered.
-$imagename has to be replace with the official name of the image you generate with the Dockerfile
+## Requirements
+### Python Libraries
+kuberentes python client: <https://github.com/kubernetes-client/python>
+Install with `pip install kubernetes` onto machine or into a fitting virtual env.
+
+Prometheus Api Client: <https://pypi.org/project/prometheus-api-client/>
+Install with `pip install prometheus_api_client`.
+
+Alternatively, install everything using `pip install -r requirements.txt`.
```

Comment:
I would suggest changing the order of install options, as typically you should install the requirements using the requirements.txt file and not manually